src/axom/mint/docs/sphinx/sections/mesh_representation.rst

```diff
@@ -0,0 +1,330 @@
+.. ##
+.. ## SPDX-License-Identifier: (BSD-3-Clause)
+
+.. _sections/mesh_representation:
+
+
+.. #############################################################################
+..  MESH Representation
+.. #############################################################################
+
+.. _MeshRepresentation:
+
+Mesh Representation
+-------------------
+
+Irrespective of the mesh type, a mesh essentially provides a data structure
+that enables efficient *storage*, *management* and *access* of:
+
+#. Mesh :ref:`Geometry` information (i.e., nodal coordinates),
+
+#. Mesh :ref:`Topology` information (i.e., cell-to-node connectivity, etc.), and
+
+#. :ref:`FieldData` stored on a mesh
+
+The underlying, concrete representation of the :ref:`Geometry` and
+:ref:`Topology` of a mesh is the key distinguishing characteristic used to
+classify a mesh into the different :ref:`MeshTypes`. As a prerequisite to the
+proceeding discussion on the taxonomy of the various :ref:`MeshTypes`, this
+section provides a high level description of the key constituents of the
+:ref:`MeshRepresentation`. Namely, the :ref:`Topology`, :ref:`Geometry` and
+:ref:`FieldData` comprising a mesh.
+
+.. _Topology:
+
+Topology
+^^^^^^^^
+
+The topology of a mesh, :math:`\mathcal{M}(\Omega) \in \mathbb{R}^d`, is
+defined by the collection of topological entities, e.g., the :ref:`Cells`,
+:ref:`Faces` and :ref:`Nodes`, comprising the mesh and the associated
+*adjacency* information that encodes the topological connections between them,
+broadly referred to as :ref:`Connectivity` information.
+Each topological entity in the mesh is identified by a unique index, as depicted
+in the sample :ref:`UnstructuredMesh` shown in
+:numref:`figs/meshRepresentation`. This provides a convenient way to traverse
+and refer to individual entities in the mesh.
+
+.. _figs/meshRepresentation:
+.. figure:: ../figures/mesh_representation.png
+  :align: center
+  :scale: 95%
+  :alt: Mesh Representation
+
+  Sample unstructured mesh. Each node, cell and face on the mesh has a unique
+  index.
+
+In Mint, the three fundamental topological entities comprising a mesh are
+(1) :ref:`Cells`, (2) :ref:`Faces`, and (3) :ref:`Nodes`.
+
+.. note::
+  The current implementation does not provide first class support for edges and
+  associated edge data in 3D. However, this is a feature we are planning
+  to support in future versions of Mint.
+
+.. _Cells:
+
+Cells
+""""""
+
+A cell, :math:`\mathcal{C}_i`, is given by an ordered list of :ref:`Nodes`,
+:math:`\mathcal{C}_i=\{n_0,n_1,...n_k\}`, where each entry,
+:math:`n_j \in \mathcal{C}_i`, corresponds to a
+unique node index in the mesh. The order of :ref:`Nodes` defining a cell is
+determined according to a prescribed local numbering convention for a
+particular cell type. See :numref:`figs/linearCells` and :numref:`figs/q2Cells`.
+All Mint :ref:`CellTypes` follow the `CGNS`_ standard local numbering
```

Comment:
This needs to link to https://cgns.github.io/CGNS_docs_current/sids/conv.html#unstructgrid.  I understand that you may wish to link to CGNS as a top-level reference, but please here link directly to the node ordering conventions.